uportal-war/src/main/webapp/media/skins/respondr/defaultSkin/skin.xml

```diff
@@ -42,6 +42,7 @@
   <css included="plain" resource="true">/scripts/jstree/style.css</css>
   <css included="aggregated" resource="true">/scripts/jstree/style.min.css</css>
 
+  <css import="true">../common/common_skin.xml</css>
```

Comment:
Looks like library CSS for...

* datatables/1.9.4/plugins/integration/bootstrap/3;  and
* datatables/1.9.4/extras/column-filter-widgets

wasn't being included at all.